src/mym.cpp

```diff
@@ -52,8 +52,14 @@ typedef size_t mwIndex;
 
 //The crazy do{}while(0) constructions circumvents unexpected results when using the macro followed by a semicolon in
 //an if/else construction
-#define READ_UINT64(dst,src) do{ safe_read_64uint( (dst), (_uint64*)(src), 1 );   (src) += sizeof(_uint64);  } while(0)
-#define READ_UINT64S(dst,src,n) do{ safe_read_64uint( (dst), (_uint64*)(src), n );   (src) += (n) * sizeof(_uint64);  } while(0)
+#if DJ_32BIT_FLAG == 1
+  #define READ_UINT64(dst,src) do{ safe_read_32uint( (dst), (_uint32*)(src), 1 );   (src) += sizeof(_uint32);  } while(0)

```

Comment:
This creates confusion since the main code uses the function `READ_UINT64` to read `uint32`. I suggest using something lake `READ_UINT` or `READ_ARRAY_SIZE` since this function is exclusively used for reading array dimensions and sizes.

Reply:
Yes, I know this is not ideal. It was only an example of how this can be done. I think this PR was meant to be an example of what to do and not meant to be pushed.
Let me know if it is needed I apply the changes you mentioned.